cfme/tests/automate/test_service_dialog.py

```diff
@@ -481,3 +484,128 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.fixture(scope="module")
+def custom_button(appliance):
```

Comment:
I don't think we need Factories as fixtures here. Is it not possible to use `import_dialog` here only than sending it form test with special call?

Reply:
Yeah. got it. updated!
Thanks for this suggestion :100: